src/Components/WebAssembly/WebAssembly/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -24,6 +24,7 @@ internal sealed partial class WebAssemblyRenderer : WebRenderer
     private readonly ILogger _logger;
     private readonly Dispatcher _dispatcher;
     private readonly IInternalJSImportMethods _jsMethods;
+    private static readonly ComponentPlatform _componentPlatform = new("Webassembly", isInteractive: true);
```

Comment:
Probably should be:
```suggestion
    private static readonly ComponentPlatform _componentPlatform = new("WebAssembly", isInteractive: true);
```

Reply:
I leave it up to you and Steve. In the past we've done `Webassembly` as one word, that's why I chose that value, but I might have miss-interpreted that.

Reply:
I guess both could be legitimate but since the assembly name is `M.A.C.WebAssembly` and it's cased as [WebAssembly](https://webassembly.org/) in its official name, I'd err on the side of that.